spec/authentication.yaml

```diff
@@ -0,0 +1,191 @@
+
+components:
+  schemas:
+    LoginName:
+      type: string
+      minLength: 1
+      example: "admin"
+
+  paths:
+    Login:
+      get:
+        tags:
+        - "authn"
```

Comment:
the name of the file makes me wonder if we should just have the tag be `authentication`

is there a benefit to short tags?